src/main/scala/sigmastate/lang/SigmaParser.scala

```diff
@@ -67,11 +70,13 @@ object SigmaParser extends Exprs with Types with Core {
     case "<"  => LT(l, r)
     case "-"  => builder.mkMinus(l.asValue[SLong.type], r.asValue[SLong.type])
     case "|"  => builder.mkXor(l.asValue[SByteArray], r.asValue[SByteArray])
```

Comment:
@greenhat This symbol `|` if for bitwise OR on Numeric

Reply:
Yes, but we currently use "|" in test scripts. It goes way beyond the parser layer. I thought it could be fixed closer to the finish.

Reply:
@greenhat Let's ignore those tests for a while, but make parser level feature complete.

Reply:
Done.